src/main/kotlin/org/misarch/catalog/event/model/ProductVariantVersionDTO.kt

```diff
@@ -24,5 +25,6 @@ data class ProductVariantVersionDTO(
     val createdAt: String,
     val canBeReturnedForDays: Double?,
     val productVariantId: UUID,
-    val taxRateId: UUID
+    val taxRateId: UUID,
+    val weight: Double
```

Comment:
On second thought, shouldn't it be
```suggestion
    val weight: Double?
```
- i.e. digital products such as ebooks or images

Reply:
Currently, our shop does not support digital products at all:
- we do not have a concept of ownership
- we do not have a concept how to deliver codes etc. (or even do that somewhat securely)
- our business processes require that the items are sent to you, so we would send the digital goods to you
I'm a bit reluctant to introduce the concept "digital good" partially by just allowing the weight to be undefined.
If we really want to support digital goods (which I have nothing against, but also think that it is probably a low-priority issue), I think we should discuss this and then fully adapt the domain model and our business processes to introduce the concept